quick-start/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	vault "github.com/hashicorp/vault/api"
+)
+
+const (
+	vaultServerAddr = "http://127.0.0.1:8200"
+	token           = "dev-only-token" // This is the root token for our Vault dev server. Don't do this in production!
+	pathToSecret    = "secret/data/my-secret-password"
+	secretKey       = "password"
+	secretValue     = "Hashi123"
```

Comment:
I've been debating which I like more, having these strings as constants up here, having them constants next to where they are used in the code (to reduce the amount of scrolling), or having them hardcoded. For a quick start hard-coding feels kind of appropriate. But I think all 3 ways work OK

Reply:
I think all 3 ways are OK, but I'd be inclined to +1 the "hard code in place" to reduce scrolling. It also makes each block of code copy-pasta friendly.

Reply:
Sounds good, I'll revert back to how it is in vault-examples, just hard-coded strings